### PR TITLE
63681 Remove any references of va_online_scheduling_print_list feature toggle

### LINKED ISSUE #63681: Remove any references of va_online_scheduling_print_list feature toggle
## Description
We no longer need to reference the feature toggle `va_online_scheduling_print_list` in the frontend code. The Print List work was deployed on August 2, 2023. The product team is requesting to remove any references of the feature toggle. 

---

## Acceptance Criteria

- [ ] Remove any references of the feature toggle `va_online_scheduling_print_list`
- [ ] Delete the feature toggle `va_online_scheduling_print_list` in [staging](https://staging-api.va.gov/flipper/features/ ) and [production]( https://api.va.gov/flipper/features)
- [ ] Changes have been updated in `vets-website` and `vets-api`

### Design Assets
N/A

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] [Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/product/feature_toggles.md) is updated

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Summary
This PR removes any references of va_online_scheduling_print_list feature toggle

## Related issue(s)
fixes department-of-veterans-affairs/va.gov-team#63681

## Testing done
Unit testing.

## Screenshots
N/A

## What areas of the site does it impact?
VAOS

## Acceptance criteria
All test must pass

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx [hunk 1]
```diff
@@ -96,7 +95,7 @@ function renderWarningNotification() {
   };
 }
 
-function getSpacing({ isPrintList, isPast, isPending }) {
+function getSpacing({ isPrintList = true, isPast, isPending }) {
```
Comment (jenniemc) [inline src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx:98]: I think you can eliminate the `getSpacing` function all together and replace where it was called (at the h1 selector below)  with the classnames:
```
classNames(
      `xsmall-screen:vads-u-margin-bottom--3 small-screen:${
        isPast || isPending
          ? 'vads-u-margin-bottom--3'
          : 'vads-u-margin-bottom--4'
      }`,
    )
 ```

File: src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx [hunk 2]
```diff
@@ -193,23 +191,10 @@ export default function AppointmentsPage() {
...
-  if (featureStatusImprovement) {
-    paragraphText =
-      'Your appointment requests that haven’t been scheduled yet.';
-  }
   return (
     <PageLayout showBreadcrumbs showNeedHelp>
-      <h1 className={getSpacing({ isPrintList, isPast, isPending })}>
-        {pageTitle}
-      </h1>
+      <h1 className={getSpacing({ isPast, isPending })}>{pageTitle}</h1>
```
Comment (jenniemc) [inline src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx:196]: Once `getSpacing` function is eliminated , replace the call with the actual classnames

File: src/applications/vaos/tests/appointment-list/components/AppointmentsPageV2/index.unit.spec.jsx [hunk 3]
```diff
@@ -429,7 +429,6 @@ describe('VAOS <AppointmentsPage>', () => {
         vaOnlineSchedulingDirect: true,
         vaOnlineSchedulingCommunityCare: false,
         vaOnlineSchedulingStatusImprovement: true,
-        vaOnlineSchedulingPrintList: true,
```
Comment (jenniemc) [inline src/applications/vaos/tests/appointment-list/components/AppointmentsPageV2/index.unit.spec.jsx:432]: Consider changing the description of the Describe

**Review by jenniemc — COMMENTED** [review]
Looking good. Just some suggestions.

File: src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx:98]: Done.

File: src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx (hunk 2 above)
Comment (vbahinwillit) [reply src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx:196]: Done.

**Review by jenniemc — APPROVED** [review]
LGTM!

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**